launcher/StringUtils.cpp

```diff
@@ -212,3 +212,25 @@ QPair<QString, QString> StringUtils::splitFirst(const QString& s, const QRegular
     right = s.mid(end);
     return qMakePair(left, right);
 }
+
+QString StringUtils::htmlListPatch(QString htmlStr)
+{
+    QRegularExpression match("<w*/w*ulw*>|<w*ulw*/w*>");
+    int pos = htmlStr.indexOf(match);
+    int imgPos, dist;
+    while (pos != -1) {
+        dist = htmlStr.indexOf(">", pos) - pos + 1;  // Get the size of the </ul> tag. Add one for zeroeth index
+        pos = pos + dist;
```

Comment:
Not sure if you need a +1 or a-1 after the index of, but you do not need to calculate the size of the tag just use the indexOf(">"):
```suggestion
        pos = htmlStr.indexOf(">", pos) ;
```

Reply:
The +1 is necessary, as otherwise the position references the ">" character. I do agree with not needing dist. I don't exactly know why I kept it.